dev/src/components/__tests__/CourseCreationInfo.test.tsx

```diff
@@ -0,0 +1,20 @@
+import CourseCreationInfo from "../admin/CourseCreationInfo";
```

Comment:
Could you provide a markdown doc or add to an existing one, that explains how to run the tests. Or if we need a plugin to run tests, we should make sure to add such plugins to the list of recommended plugins. (See also issue #615 )

Reply:
Also, is `__tests__` the recommended/required naming for this folder?